docs/source/pillars/information_governance.md

```diff
@@ -2,137 +2,258 @@
+    These data should be analysed with reports and dashboards provided to guide TRE implementer’s improvements and provide re-assurance to data consumers and subjects.
   - Mandatory
 ```
 
-## Risk management
+**Electronic Quality Management System Application:** A software application or platform used to manage and automate quality management processes, including document control, corrective actions, audits, and performance tracking.
 
-_The ability of the TRE organisation to measure, forecast and evaluate risks to information._
+```{list-table}
+:header-rows: 1
+:name: tab-ig-electronic-quality-management-system-application
+
+* - Statement
+  - Guidance
+  - Importance
+* - You could use an eQMS (Electronic Quality Management System) to standardise and automate quality management tasks and workflows, and to generate quality data and reports automatically.
+  - A basis eQMS could be a set of spreadsheets or documents held in a repository which are manually maintained.
+    More mature applications will provide workflows and generate quality data through manual and automated actions.
+  - Optional
+```
+
+## Risk Management
+
+What the organisation does to ensure information risk is measured and managed to an acceptable level.
+
+**Risk Assessment:** The systematic evaluation and analysis of potential risks, threats, or vulnerabilities, including their likelihood, potential impact, and the effectiveness of existing controls or mitigation measures.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-risk-assessment
+:name: tab-ig-risk-assessment
 
 * - Statement
   - Guidance
   - Importance
 * - You must have a way to score risk to understand the underlying severity.
-  - You have a risk assesment methodology for scoring risks on multiple axes such as impact and likelihood.
+  - You have a risk assessment methodology for scoring risks on multiple axes such as impact and likelihood.
   - Mandatory
-* - You must have a process for mitigating risk using additional controls.
-  - Risks can be reduced to a level which brings it within agreed levels of appetite.
+* - You must carry out a data processing assessment for all projects requiring a TRE that are working with sensitive data.
+  - A data processing assessment is a process designed to identify risks arising out of the processing of sensitive data and to minimise these risks as far and as early as possible. This may take the form of an existing regulatory requirements such as Data Protection Impact Assessment.
   - Mandatory
-* - You must have an understanding of risk appetite.
-  - This includes understanding ownership of risk, and ability to accept risk which falls outside of the appetite should that become necessary.
+```
+
+**Risk Treatment:** The selection and implementation of strategies, controls, or measures to manage or mitigate identified risks, such as risk avoidance, risk transfer, risk reduction, or risk acceptance.
```

Comment:
Modified from https://satre-specification.readthedocs.io/en/v0.2.0/pillars/information_governance.html#risk-management , split into separate assessment and treatment sections